src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
That function is required by data-json which it will be added in GCWeb by PR https://github.com/wet-boew/GCWeb/pull/1237